nodeadm/internal/kubelet/config.go

```diff
@@ -201,9 +201,12 @@ func (ksc *kubeletSubConfig) withNodeIp(cfg *api.NodeConfig, kubeletArguments ma
 
 func (ksc *kubeletSubConfig) withVersionToggles(kubeletVersion string, kubeletArguments map[string]string) {
 	// TODO: remove when 1.26 is EOL
-	// --container-runtime flag is gone in 1.27+
 	if semver.Compare(kubeletVersion, "v1.27.0") < 0 {
+		// --container-runtime flag is gone in 1.27+
```

Comment:
Can we have this logic for all the versions and remove this configuration from kubelet-containerd.service file?